index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <div class="logo"><a href="">Johnson</a> </div>
+        <nav> <a href=""><img class="image" src="images/Union.png" alt="dropdown"></a> </nav>
+    </header>
+    <main>
+        <section class="headline">
+            <div class="supporting-text">
+                <h2>Hey There. I'm Johnson</h2>
+                <p> I'm a software developer <br /></p> 
+                <div class="description">
+                    I can help you build a product , feature or website Look through
+                    some of my work and experience! If you like what you see and have a project you need coded,
+                    don't hestiate to contact me.
+
+                </div>
+
+                
+            </div>
+            <nav class="socialmedia">
+                <a href=""><img src="images/Disabled.png" alt=""></a>
+                <a href=""><img src="images/Enabled.png" alt=""></a>
+                <a href=""><img src="images/GitHub.png" alt=""></a>
+                <a href=""><img src="images/Twitter.png" alt=""></a>
+                <a href=""><img src="images/micro.png" alt=""></a>
+            </nav>
```

Comment:
- [ ] Kindly make sure the list of icons uses a `<ul>` tag as **container** to group them not the `nav` tag, and wrap each `<a> `tag inside `<li>` per icon. 
e.g : `<ul><li><a href=""></a></li></ul>`